app/src/main/java/com/codesoom/assignment/TaskHttpHandler.java

```diff
@@ -0,0 +1,272 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+class HttpStatusCode {
+    public static final int OK = 200;
+    public static final int NotFound = 404;
```

Comment:
Java의 `static final` 상수는 대문자로만 snake_case로 작성하는 관례가 있습니다.
```suggestion
    public static final int OK = 200;
    public static final int NOT_FOUND = 404;
```